### PR TITLE
Fixing a bunch of module tests

### PR DESCRIPTION
**Pull request by KevinMenden**
* seqwish
* homer
* gffread
* sequenzautils

### CODE REVIEW

File: tests/software/minia/main.nf [hunk 1]
```diff
@@ -7,7 +7,7 @@ include { MINIA } from '../../../software/minia/main.nf' addParams( options: [:]
 workflow test_minia {
     input = [ [ id:'test' ], // meta map
               [file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true),
-              file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true)] 
+              file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true)]
```
Comment (drpatelh) [inline tests/software/minia/main.nf:10]: This hasn't been updated?

Comment (KevinMenden) [reply tests/software/minia/main.nf:10]: oh shit sorry ... there was some other stuff in my git history it seems :facepalm: 
only the last two commits were actually intentional.
damnit ...

Comment (KevinMenden) [reply tests/software/minia/main.nf]: well I just updated it anway

**Review by drpatelh — APPROVED** [review]
Thanks @KevinMenden !

**Pull request merged**